src/Bluesky.NET/ApiClients/BlueskyApiClient.cs

```diff
@@ -16,19 +16,22 @@ namespace Bluesky.NET.ApiClients;
 public partial class BlueskyApiClient : IBlueskyApiClient
 {
     private readonly HttpClient _httpClient = new();
+    private string _baseUrl = "https://bsky.social";
 
-    public async Task<Result<AuthResponse>> RefreshAsync(string refreshToken)
+    public async Task<Result<AuthResponse>> RefreshAsync(string refreshToken, string baseUrl = UrlConstants.BlueskyBaseUrl)
     {
-        var refreshUrl = $"{UrlConstants.BlueskyBaseUrl}/{UrlConstants.RefreshAuthPath}";
+        _baseUrl = baseUrl;
+        var refreshUrl = $"{baseUrl}/{UrlConstants.RefreshAuthPath}";
         HttpRequestMessage message = new(HttpMethod.Post, refreshUrl);
         message.Headers.Authorization = new AuthenticationHeaderValue("Bearer", refreshToken);
         return await PostAuthMessageAsync(message);
     }
 
     /// <inheritdoc/>
-    public async Task<Result<AuthResponse>> AuthenticateAsync(string identifer, string appPassword)
+    public async Task<Result<AuthResponse>> AuthenticateAsync(string identifer, string appPassword, string baseUrl = UrlConstants.BlueskyBaseUrl)
     {
-        var authUrl = $"{UrlConstants.BlueskyBaseUrl}/{UrlConstants.AuthPath}";
+        _baseUrl = baseUrl;
+        var authUrl = $"{baseUrl}/{UrlConstants.AuthPath}";
```

Comment:
This url is resolving to `bsky.social/<authPath>`. This is leading to an HTTP error because the url is missing `https://`.